src/static/add_tta_match_result.html

```diff
@@ -3,35 +3,112 @@
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
 <head>
-  <title>Add a match result</title>
-  <script src="https://unpkg.com/vue@latest"></script>
-  <script src="https://unpkg.com/vue-select@latest"></script>
-  <link type="text/css" rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
-  <link type="text/css" rel="stylesheet" href="/static/styles.css">
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width">
-  <script src="/static/js/http.js" async=true></script>
-  <script src="/static/js/add_tta_match_result.js" async=true></script>
+    <title>Add a match result</title>
```

Comment:
What formatter were you using? I personally prefer 2 space indent instead of 4 space indent (same as Google). Could you modify your editor to use 2 spaces indent?